regexec.c

```diff
@@ -11191,7 +11191,7 @@ S_reghop3(U8 *s, SSize_t off, const U8* lim)
  * char pos */
 
 STATIC U8 *
-S_reghopmaybe3(U8* s, SSize_t off, const U8* const lim)
+S_reghopmaybe3(pTHX_ U8* s, SSize_t off, const U8* const lim)
```

Comment:
Shouldn't this be part of a later commit?

Reply:
Yeah, I moved it to the next commit.